distribution/docs/src/main/resources/content/_managing/_configuring/configuring-nsili-endpoint.adoc

```diff
@@ -0,0 +1,27 @@
+:title: Configuring NSILI Endpoint
+:type: subConfiguration
+:status: published
+:parent: Configuring Endpoints
+:summary: Configuring NSILI Endpoint
+:order: 03
+
+== {title}
```

Comment:
✏️ In the zip you provided on the PR, this is rendering as "7.8.6.5. Configuring KML Endpoint"

Reply:
Fixed

Reply:
I'm getting errors trying to upload a new zip though